### PR TITLE
Add German i18n file.

### PR DESCRIPTION
**Pull request by mkalus**
Added de.toml

### CODE REVIEW

**Review by matsuyoshi30 — CHANGES_REQUESTED** [review]
Thanks for your contribution! I commented on a thing, so please check it.

File: i18n/de.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[newer]
+other = "Neuere Beiträge"
+[read-more]
+other = "Weiterlesen"
+[search]
+other = "Suchen..."
+[powered-by]
+other = "Powered By"
+[theme]
+other = "Vorlage"
```
[the remark this answers is not in the document]
Comment (lomion0815) [reply i18n/de.toml:12]: I set "Theme" in my de.toml ... any translation felt wrong to me ... Theme is commonly used as a technical term.
```
[older]
other = "Älter"
[newer]
other = "Neuer"
[read-more]
other = "Weiterlesen"
[search]
other = "Suchen..."
[powered-by]
other = "Powered By"
[theme]
other = "Theme"
```

[the remark this answers is not in the document]
Comment (mkalus) [reply i18n/de.toml:12]: @lomion0815: I agree, "Theme" sounds good.

**Comment by mkalus** [comment]
Closing pull request and opening new one with corrected translation file...

**Pull request closed without merging**